.gitignore

```diff
@@ -3,3 +3,5 @@
 .python-version
 .vscode
 *.code-workspace
+*.sql.gz
+dos-dump.sql.gz
```

Comment:
This line `dos-dump.sql.gz` is not needed. Please, can we you include an empty line at the end of file?